scripts/foundry/mainnet/CreateMorphoMarkets.s.sol

```diff
@@ -0,0 +1,136 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity ^0.8.12;
+
+import "forge-std/Script.sol";
+import { console } from "forge-std/console.sol";
+import { IMorpho, MarketParams } from "../../../contracts/interfaces/external/morpho/IMorpho.sol";
+import { IMorphoChainlinkOracleV2Factory, IMorphoOracle } from "../../../contracts/interfaces/external/morpho/IMorphoChainlinkOracleV2.sol";
+import "./MainnetConstants.s.sol";
+import { StdCheats } from "forge-std/Test.sol";
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+// Before running this script, ensure that the deployer has the necessary balance in all tokens to seed the markets
+contract CreateMorphoMarkets is Script, MainnetConstants, StdCheats {
```

Comment:
we are just missing the weth/usda market

Reply:
wstETH/USDA has already been created